Automattic/woocommerce-payments
# Jetpack Connection Manager Updates

Fixes #4256

#### Changes proposed in this Pull Request





This PR updates the HTTP request library in the plugin, to remove deprecated usages of the Jetpack Connection manager as described in #4256.





#### Testing instructions





**Note:** There should be no effect on functionality.

* Spin up a new JN site. You can use `botwoo` (see https://github.com/Automattic/woocommerce-payments/pull/8584#issuecomment-2039287861)
* Go to `WooCommerce` and complete the Core onboarding by clicking "Skip Guided Setup" and then selecting a country.
* Go to `WooCommerce > Status` and scroll to the `WooPayments` block. You should see the following:

<img width="1062" alt="Screenshot 2024-04-05 at 10 41 08" src="https://github.com/Automattic/woocommerce-payments/assets/11770181/c0fda47c-5f65-42ba-abb5-02b7d30b6df8">

* Go to `Payments`, click `Connect Your Store` and approve the Jetpack connection.
* When you get redirected to the onboarding, go back and then return to `WooCommerce > Status`
* You should see something like the following:

<img width="1055" alt="Screenshot 2024-04-05 at 10 44 56" src="https://github.com/Automattic/woocommerce-payments/assets/11770181/9e0ff41a-0fbc-481d-9191-1fb6188a258d">

-------------------


**Post merge**
